scripts/test-install-zip.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+STACK_VERSION=${1:-default}
+
+cleanup() {
+  r=$?
+
+  # Dump stack logs
+  elastic-package stack dump -v --output build/elastic-stack-dump/install-zip
+
+  # Take down the stack
+  elastic-package stack down -v
+
+  for d in test/packages/*/*/; do
+    (
+      cd $d
+      elastic-package clean -v
+    )
+  done
+
+  exit $r
+}
+
+trap cleanup EXIT
+
+ARG_VERSION=""
+if [ "${STACK_VERSION}" != "default" ]; then
+  ARG_VERSION="--version ${STACK_VERSION}"
+fi
+
+# Update the stack
+elastic-package stack update -v ${ARG_VERSION}
+
+# Boot up the stack
+elastic-package stack up -d -v ${ARG_VERSION}
+
+# Verify it's accessible
+eval "$(elastic-package stack shellinit)"
+
+export ELASTIC_PACKAGE_LINKS_FILE_PATH="$(pwd)/scripts/links_table.yml"
+OLDPWD=$PWD
+
+# Build packages
+for d in test/packages/*/*/; do
+  (
+    cd $d
+    elastic-package build
+  )
+done
+cd $OLDPWD
+
+# Remove unzipped built packages, leave .zip files
+rm -r build/packages/*/
+
+# Install packages and verify
+for zipFile in build/packages/*.zip; do
+  PACKAGE_NAME_VERSION=$(basename ${zipFile} .zip)
+
+  # check that the package is installed
```

Comment:
Do this comment belong to the curl below?

Reply:
Yes, moving it to where it should be, thanks!